pkgs/by-name/sp/spec-kit/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "spec-kit";
+  version = "0.0.79";
+
+  src = fetchFromGitHub {
+    owner = "github";
+    repo = "spec-kit";
+    rev = "e6d6f3cdee99752baee578896797400a72430ec0";
+    hash = "sha256-A5WQ6/YeEfYrGRxO/V7grKB3O2wv4WIXBvNBAYxAx4Y=";
+  };
+
+  pyproject = true;
+
+  nativeBuildInputs = with python3.pkgs; [
+    hatchling
+  ];
+
+  propagatedBuildInputs =
+    with python3.pkgs;
+    [
+      typer
+      rich
+      httpx
+      platformdirs
+      readchar
+      truststore
+    ]
```

```suggestion
    (with python3.pkgs;
    [
      typer
      rich
      httpx
      platformdirs
      readchar
      truststore
    ])
```
only with parens you can separate the `with` scope

---

forgot this one